app/views.py

```diff
@@ -81,11 +81,33 @@ def progressreport():
     """Renders the report page."""
     return render_template('report.html', title=' | Progress Report')
 
-@app.route('/create', methods=['GET', 'POST'])
-def create():
-    """Renders the create page for educators."""
-    form = CreateQnForm()
-    return render_template('create.html', title=' | Create', form=form)
+@app.route('/createclass', methods=['GET', 'POST'])
+def createclass():
+    """Renders the create class page for educators."""
+    form = CreateClass()
+    if form.validate_on_submit():
+        Group.name = form.className.data
+        db.session.commit()
+        return redirect(url_for('createclasssuccess'))
+    return render_template('createclass.html', title=' | Create Class', form=form)
+
+@app.route('/createclasssuccess', methods=['GET'])
+def createclasssuccess():
+    """Renders the create class was a success page for educators."""
+    form = CreateClass()
+    return render_template('createclasssuccess.html', title=' | Create Class', form=form)
+
+@app.route('/createquiz', methods=['GET', 'POST'])
+def createquiz():
+    """Renders the create quiz page for educators."""
+    form = CreateQuiz()
+    return render_template('createquiz.html', title=' | Create Quiz', form=form)
+
+@app.route('/createquizsuccess', methods=['GET', 'POST'])
+def createquizsuccess():
```

Comment:
Similar here as createclasssuccess. Just go straight to where the user would need to go. Also it should only allow GET requests.

Reply:
Educator might wanna create another quiz, or preview their quiz, or go back to their dashboard

Reply:
To preview the newly created quiz you would need it as input to the page.

Reply:
Need an explanation, is there a link of sorts or is it anywhere in our existing resources?

Reply:
The preview will need to display the questions they just inputted. I think possibly 
- they add a bunch of questions
- it gets handled by JavaScript, maybe added to a JSON
- each click of add next question does not add to database yet
- when submit quiz, then the JSON will get submitted to the app which will then process it and add to the database
- the same JSON can be used for previewing the quiz
- preview might not even route to another page
- as long as they have not left the page or refreshed the preview is available
- otherwise any view of the quiz - from the dashboard - would need a query to the database

Reply:
Looks like you'll be going for add to database directly instead of using JSON. In that case, having something similar after creating a class could be used.
- Preview Quiz (Query all questions in the quiz that was just created). This could just be the edit questions feature in fact, in case they want to change the questions.
- Create a New Quiz (Might not even be needed)
- Dashboard

The link for preview quiz needs have the quizID tagged to it so it leads to the right quiz.